src/components/Company/CompanyName/CorrectNameToNumber.vue

```diff
@@ -0,0 +1,62 @@
+<template>
+  <v-form lazy-validation>
+    <v-layout row>
+        <v-checkbox
+          v-model="correctToNumbered"
+          label="Change the company name to 1234567 B.C. Ltd."
```

Comment:
Is this a hardcoded value or is it supposed to be {identifier} B.C. Ltd.

Reply:
I didn't even think of this until you guys mentioned it :) I have updated to be populated by the Companies Business identifier in the next commit!